tests/src/test_cases/test_anim_timeline.c

```diff
@@ -322,4 +322,77 @@ void test_anim_timeline_reverse(void)
     TEST_ASSERT_EQUAL(20, lv_obj_get_x(obj));
 }
 
+
+void anim1_exec_cb(void * var, int32_t v)
```

Comment:
Hi,
Could you please fix the warnings below? You can use `LV_UNUSED(var);`

Reply:
"I apologize, I'm a newbie with these GitHub tools. 
When you mention 'LV_UNUSED(var)', 
do you mean I should add it like this?

![image](https://github.com/lvgl/lvgl/assets/15370479/a5f88884-ab1d-4d64-b2b6-a5795c49477d)

Reply:
Yes. You are doing it correct. Now the warnings in this page https://github.com/lvgl/lvgl/pull/6085/files is gone.
I'm not sure how to test the new feature `completed_cb` appropriately. The test case you need some assert.

Probably you can `lv_test_wait` some time and then check if a global variable is set in `completed_cb` or not.